.github/workflows/intergate.yml

```diff
@@ -0,0 +1,17 @@
+name: Test the app
+
+on:
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  test_pull_request:
+    runs_on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+      - run: npm run test
```

Comment:
these commands do not seem to relate to commands in `package.json`?

Reply:
@eddiejaoude sir,

This PR is directly on **master** branch. But When I open PR there was not any branch for `Reactjs`. 
You can close it. and create a new branch so that I can give PR on that branch.

- **intregate.yml** file is actually for github actions. In future, if some one want to contribute to `Reactjs` branch. then his code have to pass the test written in file [App.test.tsx](https://github.com/vivek80801/rock-paper-scissors-game/blob/rock-paper-scissor-game-with-reactjs-and-typescript/src/App.test.tsx).

- I had provided screenshots in [README](https://github.com/vivek80801/rock-paper-scissors-game/blob/rock-paper-scissor-game-with-reactjs-and-typescript/README.md)

- I had also provided instructions how to set up project in [CONTRIBUTING.md](https://github.com/vivek80801/rock-paper-scissors-game/blob/rock-paper-scissor-game-with-reactjs-and-typescript/CONTRIBUTING.md).

- I think Github templates means Github ISSUE TEMPLATE. those files are default files that github provide for ISSUE_TEMPLATES. I had just copy pest 😅. 

- `npm ci` is for clean insallation of dependency. it is equivalent to `npm install`

- `npm run test` this command run test runner `jest` and `react-testing-library` for testing app. Most people says test driven development(TDD) helps in producing code faster and efficient.

Reply:
> This PR is directly on master branch. But When I open PR there was not any branch for Reactjs.

I created a new react branch 👍

Reply:
I think. we are ready to merge pull request.

Reply:
Ok lets merge 👍 - more improvements and tweaks can come later in the `react` branch